pandas/core/frame.py

```diff
@@ -518,7 +518,17 @@ def _get_axes(N, K, index=index, columns=columns):
             return _arrays_to_mgr([values], columns, index, columns,
                                   dtype=dtype)
         elif is_datetimetz(values):
-            return self._init_dict({0: values}, index, columns, dtype=dtype)
```

Comment:
I think we we can just call ``_arrays_to_mgr`` here instead, w/o all of this additional code